Subamarines.md

```diff
@@ -8,32 +8,30 @@ end-to-end trading of digital assets in trustless and decentralised manner.
 
 ## Approach
 
-Submarine Swaps lets users send Lightning payments to a middleman on the Lightning Network; that middleman will send a corresponding amount of bitcoin to a regular (on-chain) Bitcoin address. It also works the other way around: users can send regular on-chain payments to the middleman; that middleman will then send a corresponding amount of bitcoin to a receiving Lightning node on the Lightning Network. A submarine swap has an on-chain side that can be on virtually any chain, so the altcoins don’t need to have SegWit or an lightning implementation for this to work. Go Doge! Submarine swaps between on-chain LTC to off-chain BTC haven been successfully [tested](https://twitter.com/alexbosworth/status/1025168088595984384) recently by Alex Bosworth.
+Submarine Swaps let users send Lightning payments to a middleman on the Lightning Network; that middleman will send a corresponding amount of Bitcoin to a regular (on-chain) Bitcoin address. It also works the other way around: users can send regular on-chain payments to the middleman; that middleman will then send a corresponding amount of Bitcoin to a receiving Lightning node on the Lightning Network. A Submarine Swap has an on-chain side that can be on virtually any chain, so the altcoins don’t need to have SegWit or a Lightning implementation for this to work. Go Doge! Although there is a middleman involved Submarine Swaps are completely trustless and can get refunded if the middleman is dishonest and tries to cheat.
+
+Submarine swaps between on-chain LTC to off-chain BTC haven been successfully [tested](https://twitter.com/alexbosworth/status/1025168088595984384) recently by Alex Bosworth.
 
-Apart from aforementioned on-chain to off-chain swap, it is also possible to implement off-chain to on-chain swaps. Both being hooked up to one other, will result in on-chain to on-chain swaps. This will allow users to swap funds without needing a second layer wallet. Two submarine swaps tangled together can make any permutations of on-chain and off-chain swaps possible. We propose to utilise this novel technology to help us achieve our goal of connecting digital asset exchanges in Exchange Union ecosystem.
+In conclusion Submarine Swaps allow users to swap funds on-chain to off-chain, off-chain to on-chain, and tangled together on-chain to on-chain. Users would not need a second layer wallet. Although Submarine Swaps can make any permutations of on-chain and off-chain swaps possible. We propose to utilise this novel technology to help us achieve our goal of connecting digital asset exchanges in Exchange Union ecosystem.
```

Comment:
`Although Submarine Swaps can make any permutations of on-chain and off-chain swaps possible` , comma?

Reply:
No just remove the `Although`. I want to say that users can use their second layer wallets but don't have to because make every permutation of on-chain and off-chain swaps possible.